### DISCUSSION TITLE
Cannot select multiple devices when trying to model breakout cable

### DISCUSSION THREAD

**Discussion by postilion** [netbox-community/netbox#17017] [Q&A]
Using NetBox v4.0.3, I've been beating my head against the wall trying to model 100/25G and 40/10G Passive DAC Breakout cables from a Juniper interface to various devices.  In each case a single breakout cable is used to feed three or four interfaces on separate devices.

For example:
|device-a|port-a|device-b|port-b|
|-------------|-------------|-------------|-------------|
|juniper|et-0/0/9|rubrik-a|D2|
|juniper|et-0/0/9|rubrik-b|D2|
|juniper|et-0/0/9|rubrik-c|D2|
|juniper|et-0/0/9|rubrik-d|D2|

I've seen plenty of examples and even the demonstration video from when the feature was introduced, but many of these assume the same device on the B end, or don't directly address how multiple devices are selected.

Trying to do this via Cable Add, or by selecting the Add Cable button on an interface, and I get a dialog like this:
![image](https://github.com/user-attachments/assets/73d26d67-9d3b-4a6c-9bcd-abd09249c64a)

While the dialog _does_ allow one to add multiple interfaces, I cannot find any way to add multiple devices.

Can anyone help?  Thanks in advance,
-nic

**Comment by candlerb** [comment]
It's simpler than you think. Add one interface as shown. Use the selector button to change the device and select an interface from that other device. Repeat as required.

Unfortunately, if you add device A interface `et-1/0/9` followed by device B interface `et-1/0/9` then you'll see two "et-1/0/9": but that is correct. They are two different interfaces. They are *not* interfaces on the "Device" field that is shown above them. See #10841 - although that was for an older version of the Netbox UI, the general principle still applies.

If you can't get it to work, show the exact steps you're doing. It might be a bug in the newer UI; I have not had reason to use this functionality recently.  But best if you can update to 4.0.8 first.

**Reply by postilion** [reply]
@candlerb 
Thanks for the help, but it still doesn't work.  I end up with a cable terminated to a single interface, that being the last one selected.  I have to attend a meeting right now, but will follow up later.
-nic

**Reply by candlerb** [reply]
You're right: same issue came up in #10182, and the issue for fixing it is #16176

**Reply by postilion** [reply]
@candlerb 
Thanks for your reply, and for pointing me at the current issues!  I had tried looking in the issues, but missed those (the list is long).  I guess I'll hold my breath for a while and see what transpires.

Thanks again!
-nic
